src/main/scala/com/memsql/streamliner/examples/Transformers.scala

```diff
@@ -93,3 +90,35 @@ class JSONCheckIdTransformer extends SimpleByteArrayTransformer {
     sqlContext.createDataFrame(transformedRDD, schema)
   }
 }
+
+class TwitterHashtagTransformer extends SimpleByteArrayTransformer {
+  override def transform(sqlContext: SQLContext, rdd: RDD[Array[Byte]], config: UserTransformConfig, logger: PhaseLogger): DataFrame = {
+    val columnName = config.getConfigString("column_name").getOrElse("hashtags")
+
+    // convert each input element to a JsonValue
+    val jsonRDD = rdd.map(r => byteUtils.bytesToUTF8String(r))
+
+    val hashtagsRDD: RDD[String] = jsonRDD.mapPartitions(r => {
+      // register jackson mapper (this needs to be executed on each partition)
+      val mapper = new ObjectMapper()
+      mapper.registerModule(DefaultScalaModule)
+
+      r.flatMap(tweet => {
+        val rootNode = mapper.readTree(tweet)
+        val hashtags = rootNode.path("entities").path("hashtags")
+        if (!hashtags.isMissingNode) {
+          hashtags.elements.flatMap(node => {
+            if (node.has("text")) { Some(node.get("text").asText) }
```

Comment:
Either break this if statement up into multiple lines or just use a .filter call to remove the nodes where node.has("text") is false.